src/browser-polyfill.js

```diff
@@ -336,5 +336,7 @@ if (typeof browser === "undefined") {
     return wrapObject(chrome, staticWrappers, apiMetadata);
   };
 
-  this.browser = wrapAPIs();
+  module.exports = wrapAPIs();
```

Comment:
I mostly like that by using the `unit` grunt-umd template we didn't need to rename the global alias to be able to export it, but I'm a bit worried that we have changed the eslint config to enable the "commonjs" mode, which can detect as "valid" something that is valid in a regular commonjs module but that we probably want to be mentioned as errors (e.g. usage of the any usage of the `require` or other globals available in a nodejs module).

How about reverting the change to `.eslintrc` and opt to use `// eslint-disable-line no-undef` on these two lines?

something like:
```diff
diff --git a/src/browser-polyfill.js b/src/browser-polyfill.js
index 5d55403..f1ab55c 100644
--- a/src/browser-polyfill.js
+++ b/src/browser-polyfill.js
@@ -336,7 +336,7 @@ if (typeof browser === "undefined") {
     return wrapObject(chrome, staticWrappers, apiMetadata);
   };

-  module.exports = wrapAPIs();
+  module.exports = wrapAPIs(); // eslint-disable-line no-undef
 } else {
-  module.exports = browser;
+  module.exports = browser; // eslint-disable-line no-undef
 }
```

If we opt for this solution (grunt-umd 'unit' template and module.exports) instead of renaming the global as I was suggesting in my previous comment, I would also add an inline comment near to this code, to explain that these `module.exports` are defined in the UMD wrapper.